### PR TITLE
Issue 39696: Search reference for file breaks when renaming container

### PR DESCRIPTION
**Pull request by Sigmonia**
Fixing tests and other issues from https://github.com/LabKey/platform/pull/1041

### CODE REVIEW

**Review by labkey-susanh — APPROVED** [review]

File: core/src/org/labkey/core/CoreContainerListener.java [hunk 1]
```diff
@@ -106,14 +106,14 @@ public void propertyChange(PropertyChangeEvent propertyChangeEvent)
...
+                    //Clear old file documents as the container name may be used in paths & Urls. Issue #39696
+                    SearchService.get().reindexContainerFiles(c);
                 }
 
                 String oldValue = (String) evt.getOldValue();
                 String newValue = (String) evt.getNewValue();
-                String message = evt.container.getName() + " was renamed from " + oldValue + " to " + newValue;
-                addAuditEvent(evt.user, evt.container, message);
+                String message = c.getName() + " was renamed from " + oldValue + " to " + newValue;
+                addAuditEvent(evt.user, c, message);
```
Comment (labkey-tchad) [inline core/src/org/labkey/core/CoreContainerListener.java:111]: Why doesn't the reindexing get triggered in `SearchContainerListener`? That's where we trigger the initial crawl for new conainers (`DavCrawler.getInstance().addPathToCrawl`)

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
This isn't working for me locally. It also appears to only address folder rename but not move.

**Comment by labkey-tchad** [comment]
They still aren't getting reindexed but at least we're only clearing the index for actual files now (12 search failures instead of 15).

**Review by labkey-tchad — APPROVED** [review]
This is an improvement and should resolve failures in Sample Manager tests. We can continue work in a new branch.

**Pull request merged**